templates/web/components/bmviii_navbar.tmpl

```diff
@@ -0,0 +1,51 @@
+{{ define "bmviii navbar" }}
+
+<nav class="navbar bm-navbar bmvii_hero" role="navigation" aria-label="main navigation" style="width: 100%;">
+	<div class="navbar-brand">
+		<a class="navbar-item" href="/">
+			<img src='{{ static_path "bmviii_navbar_logo.svg" }}' alt="BoilerMake" width="40" height="52">
+		</a>
+
+		<a role="button" class="navbar-burger burger" aria-label="menu" aria-expanded="false" data-target="nav-hamburger-menu">
+			<span aria-hidden="true"></span>
+			<span aria-hidden="true"></span>
+			<span aria-hidden="true"></span>
+		</a>
+	</div>
+
+	<div id="nav-hamburger-menu" class="navbar-menu">
+		<div class="navbar-start">
+			<a class="navbar-item" href="/faq" style="color:black; font-family:Cabin; font-weight:600; font-size:18px; padding-left:5px; padding-right:5px">FAQ</a>
+			<a class="navbar-item" href="/hackers" style="color:black; font-family:Cabin; font-weight:600; font-size:18px; padding-left:5px; padding-right:5px">Hackers</a>
+			<a class="navbar-item" href="/sponsors" style="color:black; font-family:Cabin; font-weight:600; font-size:18px; padding-left:5px; padding-right:5px">Sponsors</a>
+			<a class="navbar-item" href="/about" style="color:black; font-family:Cabin; font-weight:600; font-size:18px; padding-left:5px; padding-right:5px">About</a>
+		</div>
+
+		<div class="navbar-end">
```

Comment:
Like instead of in the hamburger?

Reply:
nah like inside of the drop down menu they should be horizontal instead of vertically one on top of the other
![Screen Shot 2020-10-06 at 00 50 48](https://user-images.githubusercontent.com/10781907/95160214-994d3f80-0765-11eb-957a-43a6d0acb89e.png)